spring-examples/springboot-example/pom.xml

```diff
@@ -18,6 +18,7 @@
   <properties>
     <vertx.version>3.6.2</vertx.version>
     <java.version>1.8</java.version>
+    <jackson.version>2.9.7</jackson.version>
```

Comment:
can you add a comment that explains that this overrides the version defined by `spring-boot-dependencies` ?

Reply:
@vietj  Yes, I added the comment in the pom file. Thanks.